docs/appendices/release-notes/5.6.0.rst

```diff
@@ -146,9 +147,16 @@ Performance and Resilience Improvements
       │  └ Collect[doc.b | [b] | (b > 1)] (rows=unknown)
       └ Collect[doc.c | [c] | true] (rows=unknown)
 
+
+.. _5.6.0_admin_ops:
```

Comment:
Seems not needed? If so, I'd remove it..

Reply:
I use it here: https://github.com/crate/crate/pull/15307/files#diff-682845acf34e36c6d05534a066bbd558f7f2b4a3273bee7e923aa68c9960dd76R82 but if you think it's too much I can remove.

Reply:
Ah I see. You could also use `Administration and Operations`_ inside the same file. But no strong opinion.

Reply:
yep, changed it per your suggestion, thx!